conf/application.conf

```diff
@@ -33,7 +33,7 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModule"
 play.modules.enabled += "uk.gov.hmrc.mongo.play.PlayMongoModule"
 
-play.additional.module = "config.V2Module"
+play.additional.module = "config.V1Module"
```

Comment:
Minor but it may be worth specifying the disabled module via
`play.modules.disabled += "play.api.i18n.I18nModule"
`